src/popup/App.vue

```diff
@@ -19,12 +19,16 @@
     >
       <Header v-if="showHeader" />
 
-      <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
-        <RouterView
-          :class="{ 'show-header': showHeader }"
-          class="main"
-        />
-      </transition>
+      <div class="app-container">
+        <transition :name="$route.meta.asModal ? 'pop-transition' : 'page-transition'">
+          <Scrollable :hide-scrollbar="!$route.meta.scrollbarVisible">
```

Comment:
In my opinion the `<Scrollable>` component should not be used to scroll whole page. It was created to scroll parts of the screen.